AirSkyBoat/AirSkyBoat
# Adjust Jug Pet stat calculations to align with era values

**_I affirm:_**


## Please enter a player-facing description

Adjusted stat calculations of BST Jug Pets.  Most noticeably pets will see a boost in Evasion and Defense. (Tiberon)
PLD pets (Crab, Beetle) will see a very large defense boost.
Diremite pets wil see a very large attack boost.
Flytrap pets are no longer BLMs but have been correctly changed to WAR.  We can consider era+ing that for Horizon.

## What does this pull request do? (Please be technical)
Some calcs here: https://docs.google.com/spreadsheets/d/1hPslj-swq5TL5-B5RQ2Gxy7m2Bl8TsHbngj5Ufw2W6A/edit?usp=sharing
Assumed valid data until we have better: https://www.ffxiah.com/node/371

1. Converts Flytrap pets to WAR, not BLMs.  No idea why they were BLMs.
2. Based on the links above - found that a subset of jug pets have A rank evasion.
3. Based on the links above - found that overall pet defense was 10-25% low, applied DEFP to all pets to close the gap.

## Steps to test these changes

Ideally do testing on Horizon code base to avoid random level jug pets.

For each pet and level to verify: 
- !changejob bst xx and !additem jug_of_whatever_the_name_is
- use Call beast
- target the pet and use !getstats
- compare expected to actual - we would like to be within ~5 stat points

## Special Deployment Considerations

None

## Some Testing Data Points
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/521c861a-8b0a-4da8-888d-79224f0cb57b)
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/9d0e2dc6-40ba-4cd5-b39d-e3bc5e47259a)

![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/8b063781-8fa6-4e19-b590-dfd377e762e5)
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/c2d71183-0760-41c7-a018-ef320a46ec7e)

![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/bd37214f-2e1e-41ce-b87e-5eb8070592a6)
![image](https://github.com/AirSkyBoat/AirSkyBoat/assets/105882290/b1fcba40-1565-4cba-aea6-d6ec184c9e1e)
